sdk/src/androidTest/java/com/mapbox/maps/RenderCacheTest.kt

```diff
@@ -87,14 +87,30 @@ class RenderCacheTest {
   }
 
   @Test
-  @Ignore("TODO Uncomment after https://github.com/mapbox/mapbox-maps-android/issues/402")
   fun setInvalidRenderCache() {
     rule.scenario.onActivity {
       it.runOnUiThread {
-        val customSize = -1
-        mapView.getMapboxMap().setRenderCacheOptions(RenderCacheOptions.Builder().size(customSize).build())
+        val invalidSize = -1
+        var caughtException = false
+        try {
+          mapView.getMapboxMap().setRenderCacheOptions(RenderCacheOptions.Builder().size(invalidSize).build())
```

Comment:
nit: we can use `@Test(expected = IllegalArgumentException.class)` to verify exceptions

Reply:
Thanks, for some reason I thought it's not applicable for those tests 🤦